packages/venia-ui/lib/components/LoadingIndicator/indicator.js

```diff
@@ -2,23 +2,19 @@ import React from 'react';
 
 import defaultClasses from './indicator.css';
 import { mergeClasses } from '../../classify';
-import Image from '../Image';
-
-import logo from '../Logo/logo.svg';
+import { Loader as LoaderIcon } from 'react-feather';
+import Icon from '../Icon';
 
 const LoadingIndicator = props => {
     const classes = mergeClasses(defaultClasses, props.classes);
     const className = props.global ? classes.global : classes.root;
 
     return (
         <div className={className}>
-            <Image
-                alt="Loading indicator"
-                classes={{ image: classes.indicator }}
-                displayPlaceholder={false}
-                height="64"
-                src={logo}
-                width="64"
+            <Icon
+                src={LoaderIcon}
+                attrs={{ width: 64, height: 64 }}
```

Comment:
I just realized we didn't update this everywhere: this `Icon` should use the new `size` prop to set its size.

```jsx
<Icon
  src={LoaderIcon}
  size={64}
  classes={{ icon: classes.indicator }}
/>
```

Reply:
@supernova-at  updated to size property. :+1: